rpc/gnmi/gnmi.proto

```diff
@@ -325,7 +329,11 @@ message UpdateResult {
     REPLACE = 2;          // The result relates to a replace of Path.
     UPDATE = 3;           // The result relates to an update of Path.
   }
-  int64 timestamp = 1;    // Timestamp of generation (ns since epoch).
+  // Deprecated timestamp for the UpdateResult, this field has been
+  // replaced by the timestamp within the SetResponse message, since
+  // all mutations effected by a set should be applied as a single
+  // transaction.
+  int64 timestamp = 1 [deprecated=true];
```

Comment:
Agree with adding a timestamp for the overall Set transaction, but is it problematic to leave this field as-is instead of deprecating?  It could provide a way to return timestamps to indicate completion of individual operations within the transaction (optionally, for targets that can support it).